tests/validation/tests/v3_api/common.py

```diff
@@ -2962,3 +2917,63 @@ def update_and_validate_kdm(kdm_url, admin_token=ADMIN_TOKEN,
     kdm_refresh_url = rancher_api_url + "/kontainerdrivers?action=refresh"
     response = requests.post(kdm_refresh_url, verify=False, headers=header)
     assert response.ok
+
+
+def prepare_hardened_nodes(aws_nodes, profile, node_roles, 
+client=None, cluster=None, custom_cluster=False):
+    i = 0
+    if profile == 'rke-cis-1.4':
+        for aws_node in aws_nodes:
+            aws_node.execute_command("sudo sysctl -w vm.overcommit_memory=1")
+            aws_node.execute_command("sudo sysctl -w kernel.panic=10")
+            aws_node.execute_command("sudo sysctl -w kernel.panic_on_oops=1")
+            if "etcd" in node_roles[i]:
+                aws_node.execute_command("sudo useradd etcd")
+            if custom_cluster:
+                docker_run_cmd = \
+                get_custom_host_registration_cmd(client,
+                                                 cluster,
+                                                 node_roles[i],
+                                                 aws_node)
+                aws_node.execute_command(docker_run_cmd)
+            i += 1
+    elif profile == 'rke-cis-1.5':
+        for aws_node in aws_nodes:
+            aws_node.execute_command("sudo sysctl -w vm.overcommit_memory=1")
+            aws_node.execute_command("sudo sysctl -w kernel.panic=10")
+            aws_node.execute_command("sudo sysctl -w vm.panic_on_oom=0")
+            aws_node.execute_command("sudo sysctl -w kernel.panic_on_oops=1")
+            aws_node.execute_command("sudo sysctl -w "
+                                     "kernel.keys.root_maxbytes=25000000")
+            if "etcd" in node_roles[i]:
+                aws_node.execute_command("sudo groupadd -g 52034 etcd")
+                aws_node.execute_command("sudo useradd -u 52034 -g 52034 etcd")
+            if custom_cluster:
+                docker_run_cmd = \
+                get_custom_host_registration_cmd(client,
+                                                 cluster,
+                                                 node_roles[i],
+                                                 aws_node)
+                aws_node.execute_command(docker_run_cmd)
+            i += 1
+    time.sleep(5)
+    return aws_nodes
+
+
+def prepare_hardened_cluster(profile, kubeconfig_path):
+    if profile == 'rke-cis-1.5': 
+        network_policy_file = DATA_SUBDIR + "/default-allow-all.yaml"
+        account_update_file = DATA_SUBDIR + "/account_update.yaml"
+        items = execute_kubectl_cmd("get namespaces -A", 
+        kubeconfig=kubeconfig_path)["items"]
+        all_ns = [item["metadata"]["name"] for item in items]
+        for ns in all_ns:
+            execute_kubectl_cmd("apply -f {0} -n {1}".
+                                format(network_policy_file, ns), 
+                                kubeconfig=kubeconfig_path)
+        namespace = ["default", "kube-system"]
```

Comment:
I went through the hardening guide once more and I think the scripts there apply the default service account patch to all namespaces not only `default` and `kube-system`.

Reply:
changed code to include all ns. Verified the hardened scan still `passes`